tests/test_eman_source.py

```diff
@@ -0,0 +1,524 @@
+import os
+import shutil
+import tempfile
+from unittest import TestCase
+
+import importlib_resources
+
+import tests.saved_test_data
+from aspire.source import EmanSource
+
+
+class EmanSourceTestCase(TestCase):
+    def setUp(self):
+        centers = {
```

Comment:
I wonder if this should be a shared resource?

Reply:
I had the same thought. What would be your preference: as a text file? dummy file just containing the `{}` set?

Reply:
sure, could save as two columns of text or a pickle of the list.

Reply:
I saw you had some trouble using Pickle.  Is it something where we just need to use a certain Pickle Protocol for version portability?

Reply:
I think it may have had more to do with the fact that the pickled data was a set. My last commit just changes it to a list, preserving order. Seemed the simplest way, rather than worrying about relying on an old pickle protocol. (just read up on it--a new one was added in Python 3.8)

Reply:
One ordering was saved in a STAR file loaded by the `src_from_relion` object. The other two objects loaded text files which were populated in the script by loading the pickled set. My assumption is that in Python 3.8 pickle loading, the ordering given when iterating through this set changed compared to the previous versions, meaning that it no longer corresponded to the ordering saved in the STAR file (which was done by loading the pickled file in 3.6 and writing out the result to a STAR). I think saving as a list solves this because there's no way the ordering would change, even among protocols. I can just change this all to JSON loading if you think that's better

Reply:
The only appeal of JSON is that the file is human-ish readable text (which might help debugging if we had a deeper problem).  Not worth changing it. I don't think we have any other JSON.

Mainly I just wanted to understand what wasn't working.  The ordering problem seems like a reasonable reason why it would not have worked. Thanks!